pkg/controllers/gracefuleviction/evictiontask.go

```diff
@@ -48,8 +48,19 @@ func assessEvictionTasks(bindingSpec workv1alpha2.ResourceBindingSpec,
 }
 
 func assessSingleTask(task workv1alpha2.GracefulEvictionTask, opt assessmentOption) *workv1alpha2.GracefulEvictionTask {
+	if task.SuppressDeletion != nil {
+		if *task.SuppressDeletion {
+			return &task
+		}
+		return nil
```

Comment:
In this case, do not need to judge the timeout?

Reply:
If users set `SuppressDeletion`, then whether to delete the task depends on this field.

Reply:
thx, i get it.